console/console-init/ui/src/modules/project/ProjectPage.tsx

```diff
@@ -105,30 +106,34 @@ export default function ProjectPage() {
   };
 
   const onConfirmDeleteAll = async () => {
-    /**
```

Comment:
can we split this logic into two parts saveIoTproject and saveMsgProject if possible.